C-Plus-Plus/README.md

```diff
@@ -132,6 +132,12 @@ _add list here_
 
 ## Other
 
+- [Array Right Rotation Naive Approach](cp/Array_Right_Rotate.cpp)
```

Comment:
can we please keep only the required changes here?

Reply:
Done:)